src/BTCPayServer.Lightning.Eclair/EclairClient.cs

```diff
@@ -191,7 +191,7 @@ public async Task<string> PayInvoice(PayInvoiceRequest payInvoiceRequest, Cancel
 
 		public Task<BitcoinAddress> GetNewAddress(CancellationToken cancellationToken = default)
 		{
-			return SendCommandAsync<NoRequestModel, BitcoinAddress>("POST", new NoRequestModel(), cancellationToken);
+			return SendCommandAsync<NoRequestModel, BitcoinAddress>("getnewaddress", new NoRequestModel(), cancellationToken);
```

Comment:
This was simply broken — added the `CanGetDepositAddress` test to prevent regressions.